### ISSUE TITLE
New Feature request: UMAP

### ISSUE THREAD

**Issue by Puriney** [wguo-research/scCancer#6]
https://github.com/wguo-research/scCancer/blob/c6078eacc2bf1ad958886c39b15e50fee92df7e6/R/scAnnotation.R#L168-L170

Could you insert the runUMAP function when running Seurat?

**Comment by wguo-research** [comment]
Yes, UMAP is also one of the updates in our new version scCancer, which will be committed recently.

**Comment by Puriney** [comment]
https://github.com/wguo-research/scCancer/blob/3f9eb9e878531d588594a132f00d60d6d38506c8/R/scAnnotation.R#L227
